src/utils/fileEncoding.ts

```diff
@@ -19,14 +19,14 @@ export function convertToBase64String(file: File): Promise<string> {
   });
 }
 
-interface FileField {
+export interface FileField {
   file: File;
 }
 
 export const encodeProfileFieldFile = async (
```

Comment:
Does this need jsdoc?

Dont know how to test these functions that require file objects.